conversations/v3/configurations.yaml

```diff
@@ -698,8 +694,7 @@ components:
     ViberChannelResponses:
       type: array
       items:
-        anyOf:
-          - $ref: '#/components/schemas/ViberChannelResponse'
+        $ref: '#/components/schemas/ViberChannelResponse'
     ViberChannelResponse:
       type: object
       allOf:
```

Comment:
[On the web, the inherited properties are not rendered.](https://api.tyntec.com/reference/conversations/current.html#viber-viberchannelresponse) Do you have an idea how to achieve that?